use_cases/eluc/app/components/context/context.py

```diff
@@ -0,0 +1,127 @@
+"""
+Context component class for selecting context.
+"""
+from dash import html, dcc, Output, Input
+import dash_bootstrap_components as dbc
+import pandas as pd
+import regionmask
+
+from app.components.context.map import MapComponent
+from app.utils import EvolutionHandler
+from data import constants
+
+
+class ContextComponent():
+    """
+    Component containing map as well as dropdowns and input fields for picking a more specific context.
+    """
+    def __init__(self, app_df: pd.DataFrame, handler: EvolutionHandler):
+        self.map_component = MapComponent(app_df)
+        self.app_df = app_df
+        self.handler = handler
+        self.countries_df = regionmask.defined_regions.natural_earth_v5_0_0.countries_110.to_dataframe()
+
+    def create_label_and_value(self, label: str, value: html.Div) -> html.Div:
+        """
+        Standard dash function that pairs a label with any arbitrary value Div.
+        """
```

Comment:
This function lines up a label with a div so we reuse it for the Lat/dropdown, lon/dropdown, year/input